### DISCUSSION TITLE
Where find the devicenumber

### DISCUSSION THREAD

**Discussion by daschmidt1994** [mak-gitdev/HA_enoceanmqtt#3] [Q&A] [answered]
Hello,

a short and maybe a stupid question. But where I can find the numbers or the leters in the devicesample?
[rocker_1]
address         = 0xBABECAFE
rorg            = 0xF6
func            = 0x02
type            = 0x02

**Comment by mak-gitdev** [comment, marked as answer]
Hello,

It is indicated on your device.
<img src="https://user-images.githubusercontent.com/73297795/201653422-f5cb246b-15e1-459e-a05f-9f662098d179.png" alt="image" width="50%"/>

But no panic if you can't find it on your device, you can use HA_enoceanmqtt to find your device ID.

1. Have a look at the readme to install and start HA_enoceanmqtt if not already done.
2. Open the log file and press one button of your rocker.
3. You will have a new input in the log file looking like:
<pre>
2022-11-13 11:49:47,193 INFO: received: <b>00:2C:2B:E2</b>->FF:FF:FF:FF (-77 dBm): 0x01 ['0xf6', '0x30', '0x0', '0x2c', '0x2b', '0xe2', '0x30'] ['0x0', '0xff', '0xff', '0xff', '0xff', '0x4d', '0x0'] OrderedDict()
2022-11-13 11:49:47,195 INFO: unknown sensor: <b>00:2C:2B:E2</b>
</pre>
Voilà.

**Comment by daschmidt1994** [comment]
Thanks
this is the output.
`2022-11-14 13:19:54,074 INFO: unknown sensor: 05:19:47:25`
I'm a little bit confused where to put this numbers now in the devicesample

**Reply by mak-gitdev** [reply]
Just like this:
<pre>
[rocker_1]
address = <b>0x05194725</b>
rorg = 0xF6
func = 0x02
type = 0x02
</pre>

Note: `rocker_1` will be used to create the name of your rocker in Home Assistant. So you can replace it by another meaningful name. You just have to avoid spaces.

**Reply by daschmidt1994** [reply]
Thanks
And the rorg, func and type where i can find this numbers?

**Reply by mak-gitdev** [reply]
This is called the EEP (EnOcean Equipment Profile) of your device. 
It should be available on your device.
Or a simple search on internet of your device followed by EEP should lead you to this data.
What kind of device are you trying to integrate ?

**Reply by daschmidt1994** [reply]
ok i see the device is not supported
ETHSA – Temperature & Humidity Sensor

**Reply by mak-gitdev** [reply]
I just had a look and it will be very easy to add your device.

How did you install HA_enoceanmqtt, standalone or addon ?

**Reply by daschmidt1994** [reply]
I installed it as addon

**Reply by mak-gitdev** [reply]
Sorry didn't have time to do it yesterday. I will push something this evening.
You will see then an update available for the addon. At this moment, please update or reinstall the addon.
After that you should see entities corresponding to your device in HA.
Please then report whether the created entities are OK or require modifications.

**Reply by mak-gitdev** [reply]
For the sake of clarity, could you please close this discussion as your initial question has been answered and start a new one with "Add support for A5-04-03" as title ?
Thanks ;)
